rmw_connext_cpp/src/functions.cpp

```diff
@@ -214,6 +213,57 @@ rmw_destroy_node(rmw_node_t * node)
   return destroy_node(rti_connext_identifier, node);
 }
 
+bool
+__process_topic_name(
+  const char * topic_name,
+  bool avoid_ros_namespace_conventions,
+  char ** topic_str,
+  char ** partition_str)
+{
+  bool success = true;
+  rcutils_string_array_t name_tokens;
```

Comment:
I should have default initialized that data structure as done in the service partitions.